src/main/scala/com/ing/wbaa/gargoyle/sts/keycloak/KeycloakTokenVerifier.scala

```diff
@@ -19,7 +19,7 @@ trait KeycloakTokenVerifier extends LazyLogging {
 
   import scala.collection.JavaConverters._
 
-  def verifyToken(token: BearerToken): Option[(UserInfo, KeycloakTokenId)] = {
+  def verifyKeycloakToken(token: BearerToken): Option[KeycloakUserInfo] = {
```

Comment:
True, reason for it was that we have two types of tokens (STS and keycloak) and this may be confusing otherwise. What do you think of:

`def verifyAuthenticationToken(token: BearerToken): Option[AuthenticationUserInfo] = {`

Reply:
ok